index.html

```diff
@@ -33,224 +33,252 @@
     </header>
     <!-- Headline -->
 
-    <section class="headline-bg headline">
-      <h1 class="headline-main">
-        I'm Promise <br />
-        Glad to see you!
-      </h1>
-      <p class="headline-paragraph">
-        I’m a software developer! I can help you build a product , feature or
-        website. Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <h3 class="headline-connect">LET'S CONNECT</h3>
+    <section class="container">
+      <div class="headline-bg headline">
+        <h1 class="headline-main">
+          I'm Promise <br />
+          Glad to see you!
+        </h1>
+        <p class="headline-paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3 class="headline-connect">LET'S CONNECT</h3>
```

Comment:
- [ ] As I can see here that margin is bigger a little bit than the design Kindly Fix it to match the design

- Your design
![image](https://user-images.githubusercontent.com/58553711/177899557-062aa6c7-1671-4c18-8491-5ef2c1b08b3e.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177899609-d41c50ab-80dc-45f4-9931-00be03d8d3dd.png)

Reply:
Correction effected.
Thanks for the review.